### PR TITLE
Build manifest for net.sourceforge.atanks

### PR DESCRIPTION
**Pull request by cosimoc**
Based on the Freedesktop 1.6 SDK.

### CODE REVIEW

File: net.sourceforge.atanks.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "rename-desktop-file": "atanks.desktop",
+    "rename-icon": "atanks",
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        {
+            "name": "alsa-lib",
```
Comment (Barthalion) [inline net.sourceforge.atanks.json]: alsa-lib is already available in the SDK/runtime.

File: net.sourceforge.atanks.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                    "type": "script",
+                    "dest-filename": "autogen.sh",
+                    "commands": [
+                        "autoreconf -i"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "alsa-plugins",
```
Comment (Barthalion) [inline net.sourceforge.atanks.json]: Ditto.

File: net.sourceforge.atanks.json (hunk 1 above)
Comment (cosimoc) [inline net.sourceforge.atanks.json]: The version in the SDK is compiled without the rawmidi API, which is required by this app. That's why I rebuild a new version here.

Comment (Barthalion) [inline net.sourceforge.atanks.json]: Ah, I see. Will start a test build soon.

**Comment by Barthalion** [comment]
Test build: https://flathub.org/builds/#/builders/5/builds/253

**Comment by cosimoc** [comment]
Seems to have worked fine!

**Comment by Barthalion** [comment]
[Done](https://github.com/flathub/net.sourceforge.atanks), thank you1

**Pull request closed without merging**